transaction-history-manager/src/main/java/com/qpay/transactionhistorymanager/model/TransactionType.java

```diff
@@ -0,0 +1,6 @@
+package com.qpay.transactionhistorymanager.model;
+
+public enum TransactionType {
+    TRANSFER,
```

Comment:
For me, p2p (person to person) is a transfer, but p2b (person to business) is a payment.